EmmanuelSage/zflow
# ##166392933  setup sequelize postgres

#### What does this PR do?

- These PR setups Sequelize for the project


#### Description of Task to be completed?

- install sequelize pg pghstore and sqlite
- initialize sequelize
- setup config
- setup babel
- test connection

#### How should this be manually tested?

  - $ git clone https://github.com/EmmanuelSage/zflow.git
  - $ npm install
  - $ npm start
  - a message should display showing connection successfull

#### What are the relevant pivotal tracker stories?

- PT Story link - [#166392933](https://www.pivotaltracker.com/story/show/166392933)
